lib/blockchain/chaindb.js

```diff
@@ -46,6 +48,8 @@ class ChainDB {
     this.state = new ChainState();
     this.pending = null;
     this.current = null;
+    options.location = resolve(options.location, '../blocks');
```

Comment:
Rather than modifying the options of `ChainOptions`, I think it would be better to pass the options direct to block store:

```
this.blockStore = new FileBlockStore({
  location: resolve(options.location, '../blocks');
});
```

Another option is to have `BlockStore` be constructed in Chain and passed to `ChainDB`.